### ISSUE TITLE
index.html

### ISSUE THREAD

**Issue by DavidZaragozaGomez** [DavidZaragozaGomez/paginafinal#1]

**Comment by DavidZaragozaGomez** [comment]
`<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>David Zaragoza</title>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/bxslider/4.2.12/jquery.bxslider.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
  <script src="https://cdn.jsdelivr.net/bxslider/4.2.12/jquery.bxslider.min.js"></script>
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
  <link rel="stylesheet" href="css/style.css">
</head>
<body>
  <div id="wrapper">
    <div id="banner-wrapper">
      <header>
        <div id="header-inner">
          <a href="index.html" id="logo">
            <img src="img/pics/nvidia_logo.png" alt="">
          </a>

          <nav>
            <a href="#" id="menu-icon">
              <i class="fa fa-bars"></i>
            </a>
            <ul>
              <li>
                <a href="index.html" class="current">
                  Home
                </a>
              </li>
              <li>
                <a href="historia.html">
                  Historia
                </a>
              </li>
              <li>
                <a href="msi.html">
                  MSI
                </a>
              </li>
              <li>
                <a href="gigabyte.html">
                  Gigabyte
                </a>
              </li>
              <li>
                <a href="asus.html">
                  Asus
                </a>
              </li>
            </ul>
          </nav>
        </div>
      </header>
      
      <div id="slide-wrap">
        <section class="slider">
          <ul class="slider1">
            <li><img src="img/slider/msi/slider1.jpg" alt=""></li>
            <li><img src="img/slider/asus/slider1.jpg" alt=""></li>
            <li><img src="img/slider/gigabyte/slider1.jpg" alt=""></li>
          </ul>
        </section>
      </div>
    </div>

    
    <section class="one-third" id="skills">
      <div class="icon-wrap">
        <i class="fa fa-comment"></i>
      </div>
      <h3>Qué son</h3>
      <p>Una tarjeta gráfica es una tarjeta de expansión de la placa base del ordenador que se encarga de procesar los datos provenientes de la unidad central de procesamiento (CPU) y transformarlos en información comprensible y representable en el dispositivo de salida (por ejemplo: monitor, televisor o proyector). Estas tarjetas utilizan una unidad de procesamiento gráfico o GPU, que muchas veces se usa erróneamente para referirse a la tarjeta gráfica en sí.</p>
    </section>
    <section class="one-third">
      <div class="icon-wrap">
        <i class="fa fa-fort-awesome"></i>
      </div>
      <h3>Un poco de historia</h3>
      <p>La historia de las tarjetas gráficas da comienzo a finales de los años 1960 cuando se pasa de usar impresoras, como elemento de visualización, a utilizar monitores. Las primeras tarjetas sólo eran capaces de visualizar texto a 40x25 u 80x25, pero la aparición de los primeros chips gráficos como el Motorola 6845 permiten comenzar a dotar a los equipos basados en bus S-100 o Eurocard de capacidades gráficas. Junto con las tarjetas que añadían un modulador de televisión fueron las primeras en recibir el término tarjeta de vídeo. <a href="historia.html">Saber mas...</a></p>
    </section>
    <section class="one-third">
      <div class="icon-wrap">
        <i class="fa fa-first-order"></i>
      </div>
      <h3>Efectos gráficos</h3>
      <p>Algunas de las técnicas o efectos habitualmente empleados o generados mediante las tarjetas gráficas pueden ser; antiescalonamiento, sombreado, renderizado de alto rango dinámico (HDR), mapeado de texturas, desenfoque de movimiento, desenfoque de profundidad, destello de lente, efecto Fresnel o imagen especular (reflejo especular) y teselado</p>
    </section>

    <div class="clearfix-padding"></div>
    
    <section class="left-col" id="portfolio">
      <h3>Errores comunes</h3>
      <p>Confundir al diseñador de la GPU con la marca de la tarjeta. Actualmente en el mercado, NVIDIA y AMD (anteriormente ATi Tecnologies) son los mayores diseñadores de chips gráficos para PC. Ellos solamente se encargan de diseñar los chip gráficos (GPU). Luego, las empresas como TSMC o GlobalFoundries fabrican las GPU, y más tarde, para su venta al público son ensambladas en PCB con memorias, por: ASUS, POV, XFX, Gigabyte, Sapphire y demás ensambladoras (aunque la versión de referencia que sale primero la fábrica el diseñador y más tarde, las ensambladoras sacan sus versiones)</p>
    </section>
    <section class="sidebar">
      <img src="img/pics/nvidia-vs-amd.png">
    </section>

    <div class="clearfix-padding"></div>
  </div>
  <div class="clearfix-padding"></div>
  <footer>
    <div class="banner-wrapper">
      <div class="icon-text">
        <div class="icon-text-icon">
          <ul class="footer-nav">
            <li>
              <a href="index.html">Home</a>
            </li>
            <li>
              <a href="historia.html">Historia</a>
            </li>
            <li>
              <a href="msi.html">MSI</a>
            </li>
            <li>
              <a href="gigabyte.html">Gigabyte</a>
            </li>
            <li>
              <a href="gigabyte.html">Asus</a>
            </li>
          </ul>
        </div>
        <div class="icon-text-text">
          <ul class="social">
            <li>
              <a href="mailto:davidzaragozag@gmail.com">
                <i class="fa fa-envelope-o"></i>
              </a>
            </li>
            <li>
              <a href="http://facebook.com/harrydavidz">
                <i class="fa fa-facebook"></i>
              </a>
            </li>
            <li>
              <a href="http://twitter.com/">
                <i class="fa fa-twitter"></i>
              </a>
            </li>
            <li>
              <a href="http://youtube.com/">
                <i class="fa fa-youtube"></i>
              </a>
            </li>
            <li>
              <a href="#">
                <i class="fa fa-instagram"></i>
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </footer>
  <footer class="second-footer">
    <p>
      &copy; <a href="index.html">graficas.com</a>
    </p>
  </footer>

  
  <script>
    $(document).ready(function(){
      $('.slider1').bxSlider({
        mode: 'fade',
      });
      $('.slider2').bxSlider({
        mode: 'fade',
      });
      $('.slider3').bxSlider({
        mode: 'fade',
      });
    });
  </script>
</body>
</html>
`
